### PR TITLE
Added a new category of low code tools

### PR DESCRIPTION
**Pull request by Nikhil-Nandagopal**
Added a new category for low code tools and updated information on Appsmith

### CODE REVIEW

**Comment by Epy** [comment]
Hello
The demo link for ERPNext is the demo from Appsmith

Also why did you put duplicate Appsmith links ?

**Comment by Nikhil-Nandagopal** [comment]
@Epy thanks for catching that! I've now fixed the link.

I've listed Appsmith under different categories because the category used to be called ERP but it's now also called low code and Appsmith is an IDE

**Review by nodiscc — CHANGES_REQUESTED** [review]
Sorry for the delay

- Please remove duplicate `ApppSmith` entries since it's already being added to `Low code`
- Should be `Low-code`
- There are several "Low-code" solutions already listed, please move them to the new section

**Comment by Nikhil-Nandagopal** [comment]
@nodiscc I've made the changes

File: README.md [hunk 1]
```diff
@@ -1069,6 +1070,17 @@ Games, game servers and control panels.
...
 - [Sakai](https://www.sakaiproject.org/) - The Sakai project provides a flexible and feature-rich environment for teaching, learning, research and other collaboration. ([Demo](https://www.sakaiproject.org/try-sakai), [Source Code](https://github.com/sakaiproject/sakai)) `ECL-2.0` `Java`
 
+### Low-Code
+
+**[`^        back to top        ^`](#)**
+
+- [Appsmith](https://www.appsmith.com) - Cloud or self-hosted open-source platform to build admin panels, CRUD apps and workflows. Build everything you need, 10x faster. ([Demo](https://app.appsmith.com), [Source Code](https://github.com/appsmithorg/appsmith)) `Apache-2.0` `JavaScript`
+- [Budibase](https://www.budibase.com) - Build and automate internal tools, admin panels, dashboards, CRUD apps, and more, in minutes. Budibase is the open source alternative to Outsystems, Retool, Mendix, Appian. ([Source Code](https://github.com/Budibase/budibase)) `GPL-3.0` `Nodejs`
+- [ERPNext](https://erpnext.com) - Free and Open Source Enterprise Resource Planning (ERP). ([Demo](https://erpnext.com), [Source Code](https://github.com/frappe/erpnext)) `GPL-3.0` `Python`
```
Comment (nodiscc) [inline README.md:1079]: ERPNext should be in the REP category, not in the Low-Code category

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nodiscc) [reply README.md:1079]: ping @Nikhil-Nandagopal

**Comment by nodiscc** [comment]
1 month without reply, closing.

**Pull request closed without merging**